packages/react-components/react-charts-preview/src/components/VerticalBarChart/VerticalBarChart.tsx

```diff
@@ -945,10 +943,8 @@ export const VerticalBarChart: React.FunctionComponent<IVerticalBarChartProps> =
       YValueHover: YValueHover,
       hoverXValue: hoverXValue,
     }),
-    gapSpace: 15,
     color: color,
     legend: calloutLegend,
-    XValue: xCalloutValue,
```

Comment:
why remove this code

Reply:
this is not used anywhere within popover